### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by julienmartinlevrai**
upgrade dss dependency and remove compiler warnings

### CODE REVIEW

File: src/test/fixtures/UnRWAUrn.sol [hunk 1]
```diff
@@ -1,5 +1,9 @@
-/// UnRWAUrn.sol -- Test fixture mocking RWAUrn
-
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// UnRWAUrn.sol -- Test fixture mocking RWAUrn
+//
+// Copyright (C) 2022 Dai Foundation
```
Comment (iamchrissmith) [inline src/test/fixtures/UnRWAUrn.sol:5]: I think this should be `2021-2022`

Also, it looks like most of the other contracts are lacking a copyright, so can you add this to those (unDai probably just needs it in addition to the one that is already there)

**Review by iamchrissmith — CHANGES_REQUESTED** [review]
Do we want to wait to update dss until we add the flap rate limiter?

**Comment by julienmartinlevrai** [comment]
@iamchrissmith this update is in order to deal with dependency issues that arose when I added the MIP21 repo to dss-exec-lib. So I don’t see a reason why we should wait for the flap rate limiter. Do you see one ?

File: src/test/fixtures/UnRWAUrn.sol (hunk 1 above)
Comment (julienmartinlevrai) [reply src/test/fixtures/UnRWAUrn.sol]: done

**Comment by iamchrissmith** [comment]
> @iamchrissmith this update is in order to deal with dependency issues that arose when I added the MIP21 repo to dss-exec-lib. So I don’t see a reason why we should wait for the flap rate limiter. Do you see one ?

no i was just thinking so we wouldn't have to update this twice, but not issues going forward with this

**Review by iamchrissmith — APPROVED** [review]

**Review by brianmcmichael — APPROVED** [review]

**Pull request merged**